### PR TITLE
12.0 website sale wishlist add tour pka seb

### PR DESCRIPTION
**Pull request by seb-odoo**
When runbot green:
- [x] move the test into the existing wishlist tour
- [x] squash the commits, not forgetting to include the original PR number too

### CODE REVIEW

**Comment by seb-odoo** [comment]
@qsm-odoo runbot green, can be reviewed/merged

File: addons/product/models/product.py [hunk 1]
```diff
@@ -359,6 +359,8 @@ def write(self, values):
             # prefetched o2m have to be reloaded (because of active_test)
             # (eg. product.template: product_variant_ids)
             self.invalidate_cache()
+            # `_get_first_possible_variant_id` depends on variants active state
+            self.clear_caches()
```
[the remark this answers is not in the document]
Comment (seb-odoo) [reply addons/product/models/product.py]: invalidate is for prefetch and caches is for ormcache

File: addons/website_sale_wishlist/static/src/js/website_sale_tour_wishlist.js [hunk 2]
```diff
@@ -39,7 +40,7 @@ tour.register('shop_wishlist', {
         {
             content: "submit login",
             trigger: ".oe_login_form",
-            run: function (){
```
[the remark this answers is not in the document]
Comment (seb-odoo) [reply addons/website_sale_wishlist/static/src/js/website_sale_tour_wishlist.js]: ok

**Comment by seb-odoo** [comment]
@rdeodoo just to be safe, could you make a quick functional review of the test here?

**Comment by rdeodoo** [comment]
Everything looks fine :+1:

**Comment by seb-odoo** [comment]
@robodoo r+

**Pull request closed without merging**